index.bs

```diff
@@ -6162,6 +6166,10 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
       See [[#sctn-validating-origin]] for guidance.
     </li>
 
+1. If <code>|C|.{{CollectedClientData/crossOrigin}}</code> is present and set to [TRUE],
+    verify that the [=[RP]=] expects that this credential would have been created within an iframe
```

Comment:
In this context, which is section **7.2 Verifying an Assertion**, should we be saying _created_ within an iframe, or __used__ (for an assertion flow) within an iframe?

Reply:
Good catch, thanks!